ddtrace/appsec/ai_guard/_api_client.py

```diff
@@ -240,6 +242,9 @@ def evaluate(self, messages: List[Message], options: Optional[Options] = None) -
                         )
 
                     span.set_tag(AI_GUARD.ACTION_TAG, action)
+                    if len(tags) > 0:
```

Comment:
would tags be None? if so, it could raise 

Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    len(None)
    ~~~^^^^^^
TypeError: object of type 'NoneType' has no len()